content/en/security_platform/cloud_workload_security/workload_security_rules.md

```diff
@@ -53,11 +57,11 @@ To ensure that your policy is evaluated in-kernel for maximum efficiency, always
 - `[event_type].file.name`
 - `[event_type].file.path`
 
-`[event_type]` can be open or exec, for example.
+**Note**: `[event_type]` can be open or exec, for example.
```

Comment:
Do you think "exec" as an executable command should be in monospace here? Feel free to discard this suggestion.

```suggestion
**Note**: `[event_type]` can be open or `exec`, for example.
```